layoutlmft/layoutlmft/data/datasets/xfun.py

```diff
@@ -101,6 +101,7 @@ def _split_generators(self, dl_manager):
         ]
 
     def _generate_examples(self, filepaths):
+        last_box = None
```

Comment:
Cool. It would be better to put it on L134.

Reply:
Moved.